string target_robot
float32 distance
float32 angle
float32 rotation